backend/controllers/usersController.js

```diff
@@ -58,9 +58,9 @@ const registerUser = asyncHandler(async (req, res) => {
                 res.status(400)
                 throw new Error("invalid user data")
             }
-        }else {
+        } else {
             res.status(400)
-            throw new Error("invalid user data") 
+            throw new Error("invalid user data")
         }
     }
 
```

Comment:
And in this line 
const newUser = await User.create({....})
I want to create a file in the App inside back end server contain the new user's data 
Is it available ?

Reply:
why do you want to do that?!

Reply:
Just want to see the user's data in my code